.github/workflows/rust-ghcr.yml

```
@@ -102,6 +102,17 @@ jobs:
               ASSET="$(curl -fsS http://127.0.0.1:8080/ | sed -n 's/.*src="\(\/assets\/[^"]*\)".*/\1/p' | head -n1)"
               test -n "$ASSET"
               curl -fsS -o /dev/null -w "%{http_code}" "http://127.0.0.1:8080${ASSET}" | grep -Eq '200'
+              # Registry + workbench assets present in image.
+              curl -fsS http://127.0.0.1:8080/api/fdd/rules | tee /tmp/fdd_rules.json
+              python3 - <<'PY'
+import json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)